auth/token.go

```diff
@@ -10,8 +11,28 @@ var (
 	applicationPrefix = "A"
 	clientPrefix      = "C"
 	pluginPrefix      = "P"
+
+	randReader = rand.Reader
 )
 
+func randIntn(n int) int {
```

Comment:
I'd prefer a implementation similar to this https://github.com/traggo/server/blob/master/auth/rand/random.go it is easier to read IMO.

Reply:
I don't really like generation algorithms that make some characters to occur more likely than others:(

Guess I will make this more readable instead.

Reply:
For every token character generated, the algorithm is losing ~0.2 bits of entrophy. So for all 14 characters generated we are losing aroung 2.8 bits of entrophy in total. This is not very significant to pose direct risks, but not so negligible that we can ignore, though., in my opinion,